inc/commonitiltask.class.php

```diff
@@ -331,7 +331,7 @@ function post_updateItem($history = 1) {
             // change ticket status (from splitted button)
             $itemtype = $this->getItilObjectItemType();
             $this->input['_job'] = new $itemtype();
-            if (!$this->input['_job']->getFromDB($this->input[$this->input['_job']->getForeignKeyField()])) {
+            if (!$this->input['_job']->getFromDB($this->fields[$this->input['_job']->getForeignKeyField()])) {
```

Comment:
Are you sure you do not want the field from input when it exists?

Reply:
Since we are in `post_updateItem`, the fields are already updated with the input values : 

PUT localhost/g94x/apirest.php/TicketTask/1
{"input": { "content":"hhhllml", "tickets_id": 22}}

![image](https://user-images.githubusercontent.com/42734840/77626730-117cb200-6f46-11ea-854a-b81e08c6feb8.png)

Reply:
As it is on post_update, I guess fields get the updated value if it has been updated.